x-pack/plugins/apm/public/components/shared/Links/DiscoverLinks/DiscoverSpanLink.tsx

```diff
@@ -5,16 +5,12 @@
  */
 
 import React from 'react';
-import { SPAN_HEX_ID, SPAN_ID } from 'x-pack/plugins/apm/common/constants';
+import { SPAN_ID } from 'x-pack/plugins/apm/common/constants';
 import { Span } from 'x-pack/plugins/apm/typings/es_schemas/Span';
 import { DiscoverLink } from './DiscoverLink';
 
 function getDiscoverQuery(span: Span) {
-  const query =
-    span.version === 'v2'
-      ? `${SPAN_HEX_ID}:"${span.span.hex_id}"`
-      : `${SPAN_ID}:"${span.span.id}"`;
-
+  const query = `${SPAN_ID}:"${span.span.id}"`;
```

Comment:
I thought the "v2" schema was using the `hex_id`?

Reply:
Yes, it was. But `hex_id` has been renamed to `id` as part of the ECS changes (I know - so many changes...)

Reply:
Phew!